src/Microsoft.Extensions.Logging.AzureAppServices/Internal/SiteConfigurationProvider.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Extensions.Logging.AzureAppServices.Internal
+{
+    public class SiteConfigurationProvider
+    {
+        public static IConfiguration GetAzureLoggingConfiguration(IWebAppContext context)
+        {
+            var settingsFolder = Path.Combine(context.HomeFolder, "site", "diagnostics");
+            var settingsFile = Path.Combine(settingsFolder, "settings.json");
+
+            // TODO: This is a workaround because the file provider doesn't handle missing folders/files
```

Comment:
That's old TODO, I guess it was https://github.com/aspnet/Configuration/issues/501 I'll remove this code and re-test scenario.